posts/navarroaxel/09.resolving-git-conflicts-using-a-merge-driver.md

```diff
@@ -77,7 +77,7 @@ Now, every conflict will be resolved by our driver accepting the upstream versio
 
 ### The npm merge driver
 
-The npm team has created an [npm-merge-drive](https://github.com/npm/npm-merge-driver) which makes a smarter merge than our previous driver - which does an overwrite 😁, keeping the lock file in sync with our changes in the `package.json`. You can just install it globally in your machine without affecting your coworkers because the `~/.config/git/attributes` is used for all your cloned repositories.
+The npm team has created an [npm-merge-drive](https://github.com/npm/npm-merge-driver) which makes a smarter merge than our previous driver - which does an overwrite 😁, keeping the lock file in sync with our changes in the `package.json`. You can just install it globally in your machine without it affecting your coworkers because the `~/.config/git/attributes` is used for all your cloned repositories.
```

Comment:
```suggestion
The npm team has created an [npm-merge-drive](https://github.com/npm/npm-merge-driver) which one makes a smarter merge than our previous driver - which does an overwrite 😁, keeping the lock file in sync with our changes in the `package.json`. You can just install it globally in your machine without it affecting your coworkers because the `~/.config/git/attributes` is used for all your cloned repositories.
```